charts/terraform-operator/templates/controller-serviceaccount.yaml

```diff
@@ -1,10 +1,16 @@
+{{- if .Values.controller.serviceAccount.create }}
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: {{ .Release.Name }}
+  name: {{ default .Release.Name }}
```

Comment:
Why did you include `default` here?

Reply:
removed; wanted to add the option to overwrite, but didn't do it eventually